### ISSUE TITLE
Display token balances for contracts

### ISSUE THREAD

**Issue by AntonAndell** [sudoblockio/icon-tracker-frontend#411]
When using the tracker i usually want to see token balance of certain contracts. While manually changing from /contract/ to /address/ in the URL solves that it would be good to display tokens on a contract page as well.

**Comment by robcxyz** [comment]
Yeah this is a good suggestion. Also should be displaying ICX balance the same with staked ICX as well. 

This also relates to the /token/* pages so we should update all of that

**Comment by robcxyz** [comment]
This needs to get fixed 

https://tracker.icon.community/address/cx43e2eec79eb76293c298f2b17aec06097be606e0
https://tracker.icon.community/contract/cx43e2eec79eb76293c298f2b17aec06097be606e0

Same for token 

https://tracker.icon.community/token/cx43e2eec79eb76293c298f2b17aec06097be606e0

Though this is not a token so this is not normal... 

Also all these 

![image](https://github.com/sudoblockio/icon-tracker-frontend/assets/6512972/4b8aa0ef-b5c9-4cde-9e0e-73e7f5aade53)

Should not look all messed up

**Comment by robcxyz** [comment]
This looks like the target block height was commented out. 

https://tracker.icon.community/address/hxfc56203484921c3b7a4dee9579d8614d8c8daaf5

That data should still show here

https://tracker.v2.mainnet.sng.vultr.icon.community/address/cx43e2eec79eb76293c298f2b17aec06097be606e0

**Comment by robcxyz** [comment]
Keeping open till #443 closes
